README.md

```diff
@@ -168,6 +168,10 @@ Each line is a filter and `lino` apply it to the start table to extract data.
 
 `--limit` is applied for each line in filters file. With a filters's file of `N` lines and a limit of `L` lino could extract a maximum of `N` x `L` lines.
 
+#### --where
+
+`--where` argument is a raw SQL clause criteria applied to the start table. It's combined with `--filter` or `--filter-from-file` with the `and` operator.
```

Comment:
it might be ambiguous whether the "--where" argument needs to actually include the sql keyword "where" or not (`--where "WHERE active=1"` vs `--where "active=1"`)

Reply:
the limitations to this where clause should also be addressed here (I'm thinking of the reference to other tables for instance)

Reply:
```suggestion
`--where` argument is a raw SQL clause criteria (without `where` keyword) applied to the **start table only**. It's combined with `--filter` or `--filter-from-file` with the `and` operator.
```